internal/ackhandler/received_packet_history.go

```diff
@@ -75,10 +75,15 @@ func (h *receivedPacketHistory) ReceivedPacket(p protocol.PacketNumber) error {
 
 	// create a new range at the beginning
 	h.ranges.InsertBefore(utils.PacketInterval{Start: p, End: p}, h.ranges.Front())
-
 	return nil
 }
 
+func (h *receivedPacketHistory) maybeDeleteOldRanges() {
```

Comment:
Leave a comment here that this is a DOS defense?

Reply:
Done.